src/local.py

```diff
@@ -58,6 +59,27 @@ def get_path_to_game(self, title_id):
             # Console Spam (Enable this if you need to.)
             return None
 
+    async def get_game_size_in_bytes(self, title_id) -> Optional[int]:
+        path = self.get_path_to_game(title_id)
+        # We will add quotes if they are not present already.
+        if path[:1] != '"':
+            path = f'"{path}"'
+        find_game_size = subprocess.Popen(
+            f'chcp 65001 & dir {path} /a /s /-c', stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
```

Comment:
interesting way of finding sizes, a bit strange and I'm not sure if more optimal than just interating over files from Python. But works